packages/main/src/themes/sap_belize/SliderBase-parameters.css

```diff
@@ -9,10 +9,10 @@
 	--_ui5_slider_handle_width: 1.75rem;
 	--_ui5_slider_handle_border: solid 0.125rem #999;
 	--_ui5_slider_handle_background: rgba(var(--sapField_Background), 0.1);
+	--_ui5_slider_handle_hover_background: rgba(217,217,217,0.6);
```

Comment:
parameter?

Reply:
In UI5 specifications is with LESS calculation fade(darken(@sapUiFieldBackground, 15), 60)